hash.c

```diff
@@ -5727,12 +5727,16 @@ env_keep_if(VALUE ehash)
 }
 
 /*
- *  call-seq:
- *     ENV.slice(*keys) -> a_hash
- *
- *  Returns a hash containing only the given keys from ENV and their values.
+ * call-seq:
+ *   ENV.slice(*names) -> Hash
```

Comment:
The return value is not `Hash` itself.

Reply:
There are many other such, including Hash, Array, Integer, Enumerator, etc.  The call-seq returns are among the least consistent things here.   They probably should be revised wholesale, after a discussion of how they should be.

Meanwhile, what would you like this one to say?